src/Nri/Ui/SegmentedControl/V10.elm

```diff
@@ -0,0 +1,305 @@
+module Nri.Ui.SegmentedControl.V10 exposing
+    ( Option, view
+    , SelectOption, viewSelect
+    , Width(..)
+    )
+
+{-| Changes from V9:
+
+  - hides non-displayed content rather than fully removing from the DOM, allowing for the segmented control to be used as a sticky-header that doesn't lose scroll position
```

Comment:
maybe this is a nitpick, but I haven't seen "sticky" used this way before. Wouldn't we say something is "scroll-preserving" if we do this? To me, sticky means that it will stick in it's position on the screen (like `position: sticky`.)